qutebrowser/browser/commands.py

```diff
@@ -383,17 +383,18 @@ def _parse_url_input(self, url):
                 yield parsed
 
     @cmdutils.register(instance='command-dispatcher', scope='window')
-    def tab_clone(self, bg=False, window=False):
+    def tab_clone(self, bg=False, window=False, private=False):
         """Duplicate the current tab.
 
         Args:
             bg: Open in a background tab.
             window: Open in a new window.
+            private: If the tab should be detached into a private instance.
```

Comment:
```suggestion
            private: Open in a new private window.
```

to be consistent with the descriptions above.